tests/rt.conf

```diff
@@ -26,6 +26,10 @@ RUN     | cpld_restart_c384_p7
 COMPILE | -DAPP=S2S -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1                                                 | - wcoss_cray                            | fv3 |
 RUN     | cpld_debug_p7                                                                                                           | - wcoss_cray                            | fv3 |
 
+# Add aerosols (temporary)
+COMPILE | -DAPP=S2SW -DUFS_GOCART=ON -DCCPP_SUITES=FV3_GFS_v16_coupled_nsstNoahmpUGWPv1                                           | + hera.intel                            | fv3 |
+RUN     | cpld_bmark_p7_aero                                                                                                      | + hera.intel                            | fv3 |
```

Comment:
I suggest to change the test name to cpld_bmark_p7.2

Reply:
Since we are moving forward to P8 should this be cpld_bmkark_p8_aero instead or do you want this to strictly follow p7.2?

Reply:
Unless the entire feature test suite for P8 gets updated in this PR (meaning all resolutions), I consider this a feature test for s2sw+aerosols, outside of the Prototype feature set. So my suggestion is that this test is cpld_bmark_aero or something similar.